TYPO3.Flow/Classes/TYPO3/Flow/Cache/Backend/SimpleFileBackend.php

```diff
@@ -257,14 +252,7 @@ public function remove($entryIdentifier)
 
         $pathAndFilename = $this->cacheDirectory . $entryIdentifier . $this->cacheEntryFileExtension;
 
-        try {
-            $lock = new Lock($pathAndFilename);
-            unlink($pathAndFilename);
-            $lock->release();
-        } catch (\Exception $exception) {
-            return false;
-        }
-        return true;
+        return @unlink($pathAndFilename);
```

Comment:
The reason for locking was to prevent concurrent reading on this file, which has happened in the past.